pom.xml

```diff
@@ -83,6 +83,11 @@
         <artifactId>jackson-dataformat-yaml</artifactId>
         <version>2.14.0</version>
       </dependency>
+      <dependency>
+        <groupId>com.fasterxml.jackson.datatype</groupId>
```

Comment:
Do you still need those POM update?

Reply:
Only for the test, but I'm going to move it into my upcoming `cos-renderer` PR